jazz-build-module/scm-module.groovy

```diff
@@ -164,13 +164,16 @@ def setRepoPermissions(repo_owner, repo_name, admin_group) {
 }
 
 def addWebhook(repo_name, webhookName,scm_webhook_target_url) {
+    def scm_events_webhook_url = "https://${scm_config.API.PROD['*']}/api/jazz/scm-web-hook"	
```

Comment:
this line of code should be in service onboarding. Moreover use utils to grab the API Id (as we have done in other places).

Reply:
Fixed